kaniko.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: kaniko
+  version: 1.10.0
+  epoch: 0
+  description: Build Container Images In Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/GoogleContainerTools/kaniko
+      tag: v${{package.version}}
+      expected-commit: 43d2eafcf210c6ff220e9cfad52bb674f6d27fd8
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      make out/executor
+      install -m755 -D out/executor "${{targets.destdir}}"/usr/bin/executor
```

Comment:
I think the file path is important here: https://github.com/GoogleContainerTools/kaniko/blob/83311d7a8f31928e315baae541d4cea4c4a926a2/deploy/Dockerfile#L67

Iirc it has to be /kaniko

Reply:
should that go into compat package ?

Reply:
This isn't a compat issue AIUI. Compat means "it should work anywhere, but helm chart assumes it's at <path>"

In kaniko's case *kaniko* expects to be run at that location, and anything else will result in failure.

There was work to make kaniko able to run from anywhere (so it could build itself) but it petered out and nobody maintains it anymore.